server/vulnerabilities/cpe_translations.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "software": {
+      "bundle_identifier": ["us.zoom.xos"],
+      "source": ["apps"]
+    },
+    "filter": {
+      "product": ["zoom", "meetings"],
+      "vendor": ["zoom"],
+      "target_sw": ["macos", "mac_os"]
```

Comment:
Why is this `target_sw` needed here but not for the other two entries (ruby and node) below?